sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubOptions.cs

```diff
@@ -88,6 +111,55 @@ public int MaxBatchSize
         /// </summary>
         public InitialOffsetOptions InitialOffsetOptions { get; }
 
+        /// <inheritdoc cref="EventProcessorOptions.TrackLastEnqueuedEventProperties"/>
+        public bool TrackLastEnqueuedEventProperties
+        {
+            get => EventProcessorOptions.TrackLastEnqueuedEventProperties;
+            set => EventProcessorOptions.TrackLastEnqueuedEventProperties = value;
+        }
+
+        /// <inheritdoc cref="EventProcessorOptions.PrefetchCount"/>
+        public int PrefetchCount
+        {
+            get => EventProcessorOptions.PrefetchCount;
+            set => EventProcessorOptions.PrefetchCount = value;
+        }
+
+        /// <inheritdoc cref="EventProcessorOptions.PrefetchSizeInBytes"/>
+        public long? PrefetchSizeInBytes
+        {
+            get => EventProcessorOptions.PrefetchSizeInBytes;
+            set => EventProcessorOptions.PrefetchSizeInBytes = value;
+        }
+
+        /// <inheritdoc cref="EventProcessorOptions.Identifier"/>
+        public string Identifier
+        {
+            get => EventProcessorOptions.Identifier;
+            set => EventProcessorOptions.Identifier = value;
+        }
+
+        /// <inheritdoc cref="EventProcessorOptions.MaximumWaitTime"/>
+        public TimeSpan? MaximumWaitTime
+        {
+            get => EventProcessorOptions.MaximumWaitTime;
+            set => EventProcessorOptions.MaximumWaitTime = value;
+        }
+
+        /// <inheritdoc cref="EventProcessorOptions.PartitionOwnershipExpirationInterval"/>
```

Comment:
We probably should consider including `LoadBalancingStrategy` as well;  it relates to these two intervals.